mep/people/templates/people/borrowing_activities.html

```diff
@@ -0,0 +1,95 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+{% load static %}
+
+{% block page-subtitle %}{{ member.firstname_last }} Membership Activity · Library Members · {% endblock %}
+
+{% block js %}
+{% render_bundle 'activities' 'js' %}
+{% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=member.firstname_last style="detail" %}
+{% endblock %}
+
+{% block main-style %}tabbed white{% endblock %}
+
+{% block content %}
+{% include 'snippets/breadcrumbs.html' %}
+<nav class="tabs" aria-label="sections">
+    <li class="tab">
+        <a href="{% url 'people:member-detail' slug=member.slug %}">biography</a>
+    </li>
+    <li class="tab" aria-selected="true">
+        <a href="{% url 'people:membership-activities' slug=member.slug %}">activities</a>
+    </li>
+    <li class="tab pending">
+        <span>cards</span>
+    </li>
+</nav>
+
+<nav class="tabs subsection" aria-label="Activities">
+    <li class="tab" aria-selected="true">
+        <a href="{% url 'people:borrowing-activities' slug=member.slug %}">Borrowing</a>
+    </li>
+    <li class="tab">
+        <a href="{% url 'people:membership-activities' slug=member.slug %}">Membership</a>
+    </li>
+</nav>
+
+<div class="activities">
+    {% comment %} preliminary form controls for sorting and expand/collapse on mobile
+    <form id="activity-mobile-control">
+        <label class="sort">
+            <span>SORTED BY</span>
+            <select>
+                <option value="1">Type</option>
+                <option value="2">Category</option>
+                <option value="3">Duration</option>
+                <option value="4" selected="selected">Start Date</option>
+                <option value="5">End Date</option>
+                <option value="6">Amount</option>
+            </select>
+            <img class="dropdown icon" src="{% static 'img/icons/chevron_down.png' %}" alt="">
+        </label>
+        <button id="expand">Expand all</button>
+    </form>
+    {% endcomment %}
+
+    {% if not event_list.count %}
+    <p>No documented borrowing activity.</p>
+    {% else %}
+    <div class="wrapper">
+    <table id="activity-table">
+        <thead>
+            <tr>
+                <th>Title</th>
+                <th>Author</th>
+                <th>Publication Date</th>
+                <th>Status</th>
+                <th data-sort-default>Start Date</th>
+                <th>End Date</th>
+            </tr>
+        </thead>
+        {% for event in event_list %}
+        <tr>
+            <td class="title">{{ event.work.title }}</td>
+            <td class="author">{{ event.work.author_list }}</td>
+            <td class="pubdate">{{ event.work.year }}</td>
+            <td class="status">
+                {% if event.borrow %}{{ event.borrow.status }}
+                {% elif event.purchase %}Purchased{% endif %}
+            </td>
+            <td class="start" data-sort="{{ event.start_date|date:"Ymd" }}">
```

Comment:
I think we need to use `partial_start_date` and `partial_end_date` here - I noticed some 1900s on Gertrude Stein's page. Although not sure what that means for the human readable display.

(Wasn't a problem for the membership activities page since the logbooks & data from them all have years associated)